lib/provider/boku.py

```diff
@@ -33,6 +33,10 @@ def create(self, request):
         cleaned = form.cleaned_data
         transaction = cleaned['param']
 
+        # Check the signature shows it was from Boku.
+        log.info('Checking signature')
+        check_sig(request.DATA)
```

Comment:
Thats a good call on the test. Its a pretty critical error imho, but yeah a 400 is a good point. I'll check how it goes through the rest of the stack.